muskathlon/migrations/10.0.3.3.0/pre-migration.py

```diff
@@ -0,0 +1,27 @@
+# -*- coding: utf-8 -*-
+##############################################################################
+#
+#    Copyright (C) 2019 Compassion CH (http://www.compassion.ch)
+#    Releasing children from poverty in Jesus' name
+#    @author: Nicolas Badoux <n.badoux@hotmail.com>
+#
+#    The licence is in the file __manifest__.py
+#
+##############################################################################
+
+
+from openupgradelib import openupgrade
+
+
+@openupgrade.migrate(use_env=True)
+def migrate(env, version):
+    if not version:
+        return
+
+    for registration in env['event.registration'].search([]):
```

Comment:
I would limit the search to registrations that have no medical survey attached.
```suggestion
    for registration in env['event.registration'].search([
            ('survey_id', '=', False),
            ('event_id.medical_survey_id', '!=', False)]):
```